config/deploy.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+# config valid for current version and patch releases of Capistrano
+lock '~> 3.17.1'
+require File.expand_path('./environment', __dir__)
+
+set :use_sudo, false
+
+# If staging_case is set to "testing", capistrano deploys Commonwealth-public-interface to "testing" server.
+# switch :stage_case to "staging" when deploying Commonwealth-public-interface to staging enviroment
+# switch :stage_case to "qc" when deploying Commonwealth-public-interface to QC server
+set :stage_case, 'qc'
```

Comment:
Is there a way to set this value without hard-coding it in this file? Possible via some arguments passed to the `cap deploy` command?